source/octf/proto/statistics.proto

```diff
@@ -81,7 +81,7 @@ message IoStatisticsEntry {
     /**
      * Metrics
      */
-    repeated IoStatisticsMetric metrics = 11;
+    map<string, IoStatisticsMetric> metrics = 11;
```

Comment:
Isn't there redundancy now - we hold metric name in metric and in the map? Isn't it also printed two times?